src/sass/modules/_m-form-elements.scss

```diff
@@ -75,9 +75,9 @@ button.form-button-disabled {
   // See 18F/web-design-standards#1490
   // This fixes things in the meantime.
   [type="checkbox"] {
-    height: auto !important;
+    height: 1.8rem !important;
```

Comment:
Got in! Thanks @ayaleloehr!

In the messaging, the checkbox in advanced search looks like this originally (with `auto`):

---
![image](https://cloud.githubusercontent.com/assets/12970166/23042136/56809164-f44c-11e6-9787-1d65317e48f6.png)

---
And with width and height to `1.8rem`:

---
![image](https://cloud.githubusercontent.com/assets/12970166/23042169/77808144-f44c-11e6-8964-374146978d02.png)

---
I think we should be golden.